c/clickhouse/clickhouse_ubi8.5.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : ClickHouse
+# Version       : Latest(Top of Tree)
+# Source repo   : https://github.com/ClickHouse/ClickHouse
+# Tested on     : UBI 8.5
+# Language      : C,C++
+# Travis-Check  : True
```

Comment:
given that this can not currently run on travis due to the build time limit being exceeded. let's set this to false. Let's add comments to explain the same here

Reply:
Done